onnxruntime/core/providers/openvino/ov_versions/capability_2021_1.cc

```diff
@@ -163,6 +164,7 @@ bool IsOpSupported(std::string name, std::string device) {
     "Tan",
   };
   std::set<std::string> supported_ops_vpu = {
+    "ArgMax",
```

Comment:
Added this op for VPU to fully support MobilenetV2-SSDLite model on UEP and with this fix, the model is working on UEP with OpenVINO Latest master